akka-projection-grpc/src/main/scala/akka/projection/grpc/consumer/ConsumerFilter.scala

```diff
@@ -214,6 +269,8 @@ object ConsumerFilter extends ExtensionId[ConsumerFilter] {
     val includeEntityOffsets3 = deduplicateEntityOffsets(includeEntityOffsets2.iterator, Map.empty)
 
     Vector(
+      if (excludeTags2.isEmpty) None else Some(ExcludeTags(excludeTags2)),
+      if (includeTags2.isEmpty) None else Some(IncludeTags(includeTags2)),
```

Comment:
`Option.when` would be nice, but it doesn't exist in Scala 2.12

Passing around empty ExcludeTags doesn't feel quite right, even though it would work. No strong reason but for example some log statements would be cluttered with criteria that are not in use.

I'll leave it as is.